src/lang-promql/complete/hybrid.ts

```diff
@@ -243,9 +237,7 @@ export class HybridComplete implements CompleteStrategy {
       }
     }
     if (includeSnippet) {
-      for (const s of parsedSnippets) {
-        options.push(s);
-      }
+      options.concat(snippets);
```

Comment:
wait @Nexucis are you sure about this? I think `concat` doesn't change the `options` array at all

Reply:
Ah yeah I forgot to get back the array returned. Thanks !

Reply:
And yeah it's maybe better to follow your way. Should cost less to keep the array and to push the element.
Thanks for the suggestion

Reply:
it's fixed. Thanks for the review @dvkndn :)